docs/workflow/blocks/context_block.rst

```diff
@@ -114,8 +114,8 @@ Security and access control
 There is currently no access control or protection in state. Any path in the global state is accessible (read and write) to any flow. 
 In the future some form of restriction may be implemented. 
 
-Subroutines
-+++++++++++
+Subroutines (Gosub)
```

Comment:
I think this might need a little more explanation - how about "Subroutines (also called Gosubs)"?

Reply:
Do you think @CodeKrakken ? I thought it was pretty clear :) But actually, I think we already do something similar with the Flow (previously called Workflow) so maybe your suggestion makes sense so we also stick to a standard. I will double-check.

Reply:
Found it: https://github.com/kendraio/kendraio-app/blob/develop/docs/index.rst?plain=1#L38
I will follow the standard